python/doc/examples/reliability_sensitivity/reliability/plot_stiffened_panel.py

```diff
@@ -0,0 +1,143 @@
+"""
+Estimate a buckling probability
+===============================
+"""
+# %%
+#
+# In this example, we estimate the probability that the output of a function
+# exceeds a given threshold with the FORM method, the SORM method and an advanced
+# sampling method.
+
+# We consider the :ref:`stiffened panel model <use-case-stiffened-panel>`.
+
+# %%
+# Define the model
+# ----------------
+
+# %%
+from openturns.usecases import stiffened_panel
+import openturns as ot
+import openturns.viewer as viewer
+
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# We load the stiffened panel model from the usecases module :
+panel = stiffened_panel.StiffenedPanel()
+distribution = panel.distribution
+model = panel.model
+
+# %%
+# See the input distribution
+distribution
+
+# %%
+# See the model
+model.getOutputDescription()
+
+# %%
+# Draw the distribution of a sample of the output.
+sampleSize = 1000
+inputSample = distribution.getSample(sampleSize)
+outputSample = model(inputSample)
+graph = ot.HistogramFactory().build(outputSample).drawPDF()
+_ = viewer.View(graph)
+
+# %%
+# Define the event
+# ----------------
+
+# %%
+# Then we create the event whose probability we want to estimate.
+
+# %%
+vect = ot.RandomVector(distribution)
+G = ot.CompositeRandomVector(model, vect)
+N0 = 165
+event = ot.ThresholdEvent(G, ot.Less(), N0)
```

Comment:
There are 3 errors here: the PEP8 (two times) and the fact that the threshold is a float, not an int.
```suggestion
g = ot.CompositeRandomVector(model, vect)
n0 = 165.0
event = ot.ThresholdEvent(g, ot.Less(), n0)
```

Reply:
I will rename the variables using more explicit names